### ISSUE TITLE
How to use MXnet to deal with the problem of image's super resolution?

### ISSUE THREAD

**Issue by lugui2009** [apache/incubator-mxnet#2536]
Hi,everybody.I am a MXnet beginer.I wanna use MXnet to do super resolution of image like this article http://mmlab.ie.cuhk.edu.hk/projects/SRCNN.html 

Because I am not familiar with DataIter, I refer to example of fcn-xs.In it ,the data_input is a image,and the label is also a image.But I encounters the wrong when I execute the command "model.fit(train_data)",and It doesn't work.

Is there someone who can tell me how to create the DataIter of this project,or is there someone who deal with the super-resolution problem with MXnet?

Thank you .

**Comment by phunterlau** [comment]
This issue is closed due to lack of activity in the last 90 days. Feel free to reopen if this is still an active issue. Thanks!
